phpcs/Standards/QualityAssurance/Sniffs/Generic/EmptySettingsSniff.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * QualityAssurance_Sniffs_Generic_EmptySettingsSniff.
+ *
+ * PHP version 5
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+/**
+ * Hardcoded paths to theme or modules need to be removed or replace by a call
```

Comment:
Please fix to something like Strongarm settings shouldn't be delivered when the value is empty, especially with pathauto patterns.